.github/workflows/ci-test-backend.yml

```diff
@@ -60,8 +55,11 @@ jobs:
         run: $GITHUB_WORKSPACE/golangci-lint run --config ${GITHUB_WORKSPACE}/backend/.golangci.yml ./...
         working-directory: backend/_example/memory_store
 
-      - name: submit coverage
-        run: $(go env GOPATH)/bin/goveralls -service="github" -coverprofile=$GITHUB_WORKSPACE/profile.cov
-        working-directory: backend
+      - name: Send coverage
```

Comment:
A small thing, but I would prefer to preserve the original name of the step.

Reply:
I reverted old name